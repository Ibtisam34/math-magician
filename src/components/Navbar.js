import React from 'react';
import { Link } from 'react-router-dom';

export default function Navbar() {
  return (
    <nav className="navbar--Container">
      <div className="navbar--heading">
        <h1>Math Magician</h1>
      </div>
      <div className="nav--List">
        <li><Link className="navLink" to="/">Home</Link></li>
        <li><Link className="navLink" to="Calculator">Calculator</Link></li>
        <li><Link className="navLink" to="Quote">Quote</Link></li>
      </div>
    </nav>
  );
}
