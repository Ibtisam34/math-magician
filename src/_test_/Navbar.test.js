import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(<Router><Navbar /></Router>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
