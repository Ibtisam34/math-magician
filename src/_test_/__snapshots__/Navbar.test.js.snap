// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="navbar--Container"
>
  <div
    className="navbar--heading"
  >
    <h1>
      Math Magician
    </h1>
  </div>
  <div
    className="nav--List"
  >
    <li>
      <a
        className="navLink"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="navLink"
        href="/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className="navLink"
        href="/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </div>
</nav>
`;